arangod/VocBase/Methods/Collections.cpp

```diff
@@ -342,6 +342,21 @@ Result Collections::create(TRI_vocbase_t& vocbase,
           factor = std::max(vocbase.replicationFactor(), cl.systemReplicationFactor());
         }
         helper.add(StaticStrings::ReplicationFactor, VPackValue(factor));
+#ifdef USE_ENTERPRISE
```

Comment:
Just curious whether we should need the ifdef here. 
It is also in the community edition this would be wrong, correct?
Otherwise, would the part in Graphmanger also need that?

Reply:
The community edition doesn't have SmartGraphs, and no SmartGraphs will ever be created there.
So technically it is not necessary, but it will not do any harm to remove the `#ifdef` here.

Reply:
Removed #ifdef.